src/components/FilterPanel.js

```diff
@@ -147,21 +148,26 @@ export default function FilterPanel(props) {
             include: Array.apply(null, Array(vaxTypes.length)).map((d) => true),
         });
     }, [data]);
-
+*/
     useEffect(() => {
         onChange({
+            showStaleData: (d) => {
```

Comment:
if we're never actually going to give the user the chance to filter stale or non-stale data, we should remove this. I think it's slowing the logic down because every time you mess with a filter you're re-evaluating everything to also check if it's stale again.

Reply:
ok. I'll try to move this to a higher level of filtering.